src/main/java/com/powsybl/openloadflow/sa/AcSecurityAnalysis.java

```diff
@@ -135,13 +191,25 @@ private SecurityAnalysisResult runSimulations(LfNetwork network, List<Propagated
             network.setReporter(preContSimReporter);
 
             // run pre-contingency simulation
-            AcLoadFlowResult preContingencyLoadFlowResult = new AcloadFlowEngine(context)
-                    .run();
+            AcloadFlowEngine engine = new AcloadFlowEngine(context);
+            engine.getContext().getEquationSystem();
```

Comment:
I missed this in my first review :-)
This is a workaround, and it should be fixed
As far as I understand, you do this to force the equation system creation and its connection to network though listener. 
But the right fix is to be able is equation system creation to better handle initially disabled buses and branches. (in actual code we probably suppose all are initially enabled)  so that we can create equation system from any network with some already disable elements.